internal/quaiapi/transaction_args.go

```diff
@@ -169,10 +169,7 @@ func (args *TransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int, no
 		minerTip *big.Int
 	)
 	// User specified max fee (or none), use those
-	gasPrice = new(big.Int)
-	if args.GasPrice != nil {
-		gasPrice = args.GasPrice.ToInt()
-	}
+	gasPrice = new(big.Int).Set(common.Big0)
```

Comment:
Why remove user specified gas price? If valid, please edit the comment to reflect this.

Reply:
Added a comment